src/datasources/cache/cache.first.data.source.ts

```diff
@@ -36,18 +36,20 @@ export class CacheFirstDataSource {
    * Gets the cached value behind the {@link CacheDir}.
    * If the value is not present, it tries to get the respective JSON
    * payload from {@link url}.
-   * 404 errors are cached with {@link ERROR_TTL_SECONDS} seconds expiration time.
+   * 404 errors are cached with {@link notFoundErrorTTLSeconds} seconds expiration time.
    *
    * @param cacheDir - {@link CacheDir} containing the key and field to be used to retrieve from cache
    * @param url - the HTTP endpoint to retrieve the JSON payload
    * @param networkRequest - the HTTP request to be used if there is a cache miss
    * @param expireTimeSeconds - the time to live in seconds for the payload behind {@link CacheDir}
+   * @param notFoundErrorTTLSeconds - the time to live in seconds for the error when the item is not found
```

Comment:
I choose `notFoundErrorTTLSeconds` as the name for this parameter. To stay consistent with the one we already have in place it could be `notFoundErrorExpireTimeSeconds` but I thought this was a bit too lengthy, specially adding the default: `defaultNotFoundErrorExpireTimeSeconds`. But this is open for discussion of course.

Reply:
Suggestion: `notFoundExpireTimeSeconds`. It's still a bit long but more in line with the other one.

Reply:
Yes, maybe it's better, thanks. 

I think this will also imply a change in the new environment variables names:
- `CONTRACT_NOT_FOUND_ERROR_TTL_SECONDS` -> `CONTRACT_NOT_FOUND_EXPIRE_TIME_SECONDS`
- `TOKEN_NOT_FOUND_ERROR_TTL_SECONDS` -> `TOKEN_NOT_FOUND_EXPIRE_TIME_SECONDS`

I'm totally fine with it, just commenting 🙂

Reply:
Changed in [f64c4da](https://github.com/safe-global/safe-client-gateway-nest/pull/578/commits/f64c4dad0e26f2530a411001a900ee3b255cc940)